test/test_ops.py

```diff
@@ -6,13 +6,13 @@
 from torch.testing import \
     (FileCheck, floating_and_complex_types_and)
 from torch.testing._internal.common_utils import \
-    (TestCase, is_iterable_of_tensors, run_tests, IS_SANDCASTLE, clone_input_helper, make_tensor)
+    (TestCase, is_iterable_of_tensors, run_tests, IS_SANDCASTLE, clone_input_helper, make_tensor,
+     gradcheck, gradgradcheck)
 from torch.testing._internal.common_methods_invocations import \
     (op_db, method_tests)
 from torch.testing._internal.common_device_type import \
     (instantiate_device_type_tests, ops, onlyCPU, onlyOnCPUAndCUDA, skipCUDAIfRocm, OpDTypes)
 from torch.testing._internal.common_jit import JitCommonTestCase, check_against_reference
-from torch.autograd.gradcheck import gradcheck, gradgradcheck
```

Comment:
cc @mruberry  FYI, we're changing here to use the test-specific version of gradcheck so that it will use the proper default for fast vs slow gradcheck.
It will also start using the other defaults about batched grad but the argument is always passed and so not changing anything.

Is that ok to do? Or you prefer to keep the plain gradcheck here with special logic to handle default value?

Reply:
Seems OK to me